src/components/registreringsinfo/hjelpekomponeneter/sykefravaer.tsx

```diff
@@ -0,0 +1,19 @@
+import {ElementLiMedLenke, Visning, VisningsListe, VisningsTitel} from "./hjelpere";
+import { teksterGruppeTittel} from "../tekster";
+import * as React from "react";
+import {SYKMELDT} from "../../../datatyper/registreringData";
+
+
+const Sykefravaer = (props: {type: string}) => (
```

Comment:
Tror det er bedre å være konsistent med const vs function.. en miks skaper rot i koden

Reply:
Så enten bruk functions all the way, eller bruk const a = () => {}